src/specific/ati.h

```diff
@@ -359,7 +359,6 @@ typedef struct {
     C3D_BOOL bClampS;
     C3D_BOOL bClampT;
     C3D_BOOL bAlphaBlend;
-    C3D_ETEXTILE eTexTiling;
```

Comment:
Both the documentation and the SDK header used to build GLRage include this line. Why was it removed?